Linux 2.6.26-rc1 and newer no longer define *_MASK; use X86_EFLAGS_*
instead (thanks to Colin Watson)
--- libx86.orig/lrmi.c
+++ libx86/lrmi.c
@@ -55,6 +55,18 @@
 #include "x86-common.h"
 
 #if defined(__linux__)
+#ifndef TF_MASK
+#define TF_MASK X86_EFLAGS_TF
+#endif
+#ifndef IF_MASK
+#define IF_MASK X86_EFLAGS_IF
+#endif
+#ifndef IOPL_MASK
+#define IOPL_MASK X86_EFLAGS_IOPL
+#endif
+#ifndef VIF_MASK
+#define VIF_MASK X86_EFLAGS_VIF
+#endif
 #define DEFAULT_VM86_FLAGS 	(IF_MASK | IOPL_MASK)
 #elif defined(__NetBSD__) || defined(__FreeBSD__)
 #define DEFAULT_VM86_FLAGS  (PSL_I | PSL_IOPL)
